test/libsolidity/syntaxTests/nameAndTypeResolution/typeChecking/return_tuple_not_convertible.sol

```diff
@@ -0,0 +1,5 @@
+contract test {
+    function f() public returns (uint256 r, uint8) { return ((12, "")); }
+}
+// ----
+// TypeError: (69-75): Return argument type bool is not implicitly convertible to expected type (type of first return variable) uint256.
```

Comment:
Interesting - where does the `bool` come from?

Reply:
hu.. good question.

Reply:
Was my fault. Didn't update the expectation after experimenting